pxtlib/discourse.ts

```diff
@@ -53,7 +53,7 @@ namespace pxt.discourse {
 
     export function topicsByTag(apiUrl: string, tag: string): Promise<pxt.CodeCard[]> {
         apiUrl = apiUrl.replace(/\/$/, '');
-        const q = `${apiUrl}/tags/${tag}.json`;
```

Comment:
To confirm, nothing changed on discourse side / this was just the wrong api it looks like, right? https://docs.discourse.org/#tag/Tags/paths/~1tags/get

Reply:
i've run this command before in the past so i think it used to work or they were more forgiving?